src/program/lwaftr/run/run.lua

```diff
@@ -102,7 +102,9 @@ function parse_args(args)
                   .." (valid values: flush, warn, off)")
       end
    end
-   function handlers.reconfigurable() opts.reconfigurable = true end
+   function handlers.reconfigurable()
+      io.stderr:write("Warning: --reconfigurable flag has been deprecated.\n")
```

Comment:
Can we adapt this note to somehow assure the user that --reconfigurable is always enabled so they can safely remove the flag from their scripts?  This note says that it's deprecated but not what action they need to take.